src/controllers/userController.js

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable linebreak-style */
+import https from 'https';
+import userHelper from '../helpers/userHelper';
+import models from '../models';
+
+const scope = ['r_liteprofile', 'w_member_social', 'r_emailaddress'];
+const clientId = process.env.LINKENDIN_CLIENT_ID;
+const clientSecret = process.env.LINKENDIN_CLIENT_SECRET;
+const redirect = process.env.LINKENDIN_REDIRECT_URL;
+const Linkedin = require('node-linkedin')(clientId, clientSecret, redirect);
+
+/**
+ * @description Handles Users can sign in via linkedin and google
+ * @class User
+ */
+class User {
+
+  /**
+      * @param {*} req Request object
+      * @param {*} res Response object
+      * @return {json} Returns json object
+      * @memberof User
+      */ async googleUrl(req, res) {
+    try {
+      this.auth = userHelper.createConnection();
+      const url = userHelper.getConnectionUrl(this.auth);
+      return res.status(200).send({ status: 200, url });
```

Comment:
why are you using res.send() instead of res.json()

Reply:
This has been updated